src/OrbitQt/HistogramWidget.h

```diff
@@ -22,6 +22,11 @@
 
 namespace orbit_qt {
 
+struct BarData {
+  double frequency;
+  int top;  // the y-coordinate for the top of the bar to be rendered
```

Comment:
Nit: I would rename to top_y_pos and erase the comment.

Reply:
Maybe, `top_pos` would be enough (see my comment above)?